### PR TITLE
[FLINK-17960][python][docs] Improve commands in the "Common Questions" document for PyFlink

### PR DESCRIPTION
**Pull request by SteNicholas**
## What is the purpose of the change

*Currently in "Common Questions" document, the command `$ setup-pyflink-virtual-env.sh` to run the script, which is not executable. It would be better to add download command and replace the command with `$ sh setup-pyflink-virtual-env.sh`.*

## Brief change log

  - *Modify script in `Preparing Python Virtual Environment` for "Common Questions" document with English and Chinese language.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / **docs** / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/table/python/common_questions.md [hunk 1]
```diff
@@ -33,10 +33,11 @@ You can download a [convenience script]({{ site.baseurl }}/downloads/setup-pyfli
 You can specify the version parameter to generate a Python virtual environment required for the corresponding PyFlink version, otherwise the most recent version will be installed.
 
 {% highlight bash %}
+$ curl -O {{ site.baseurl }}/downloads/setup-pyflink-virtual-env.sh
```
Comment (hequn8128) [inline docs/dev/table/python/common_questions.md:36]: Not sure why {{ site.baseurl }} is missing in the HTML file. @sjwiesman Do you know how to generate the URL automatically? Thanks.

**Comment by SteNicholas** [comment]
@sjwiesman Could you please help to answer the question @hequn8128 mentioned? And help to review the pull request.

**Comment by sjwiesman** [comment]
I’m not sure why it is not being templated. Unfortunately I am tied up with some urgent work and will not have time to look into this further before mid next week at the earliest.

**Comment by SteNicholas** [comment]
@sjwiesman Sorry for disturbing your work, I would like to confirm with you whether the value of ```{{ site.baseurl }}``` doesn't start with ```https:``.

**Comment by SteNicholas** [comment]
@sjwiesman Could you please review this pull request?

**Comment by hequn8128** [comment]
@sjwiesman Thanks a lot for the review. Merging...

**Pull request merged**